src/components/Activities/Activity3/Game/TimeAllocation.tsx

```diff
@@ -118,34 +118,36 @@ function TimeAllocation(props: TimeAllocationProps): JSX.Element {
   };
 
   return <div id={'time-container'} className={tutorialStage < TUTORIAL_END ? 'enable-blur' : ''}>
```

Comment:
can we replace `tutorialStage < TUTORIAL_END ` with `isTutorial` throughout the code? oops for not catching before
```suggestion
  return <div id={'time-container'} className={isTutorial ? 'enable-blur' : ''}>
```